### ISSUE TITLE
There is no way to see a quick list of all charged/threshold met petitions

### ISSUE THREAD

**Issue by H-Tevin** [ritstudentgovernment/PawPrints-Archive#175]
While there is a button for most signed petitions of all time (which is cool but functionally pointless), there is no way to see which petitions are current being reviewed by Student Government.

This feature would go a long way towards improving transparency instead of relying on just what SG puts out in emails or announcements. As of right now, I don't think there is any sort of way to check which petitions have been updated without manually scrolling through the whole site periodically.

Thanks!

**Comment by MoralCode** [comment]
I believe from some conversations I was part of in the previous 2 semesters I was told that SG was working on improved status reporting for the app, but im not sure where that is currently at.

Would love to see SG working on this or their other roadmap items more openly/in public so that external contributors can see what is happening and have an opportunity to potentially help out

**Comment by H-Tevin** [comment]
Agreed. It seems SG has really lost the motivation that they had about a decade ago. A lot of their projects have been abandoned and they aren't doing nearly as much as they used to.

I had thought the new administration would be better, but no luck it seems.

**Comment by MoralCode** [comment]
i mean from what ive heard a LARGE chunk of the pawprints codebase is legacy and most of the code paths dont actually run when the prod app is run

I think CSH has the same issues with their ScheduleMaker codebase  thats based on like a really old version of the angular framework.

Seems to me like the core problem is student churn. once the original devs graduate information about how things work tends to get lost, so people prefer making new things or doing rewrites over fixing whats already there. 

it also feels like a weird catch-22 where the processes of both SG and CSH are more internal, so external people have no visibility into whats happening. this causes people to see, from the outside, an organization that "doesn't care", and thus they conclude that its not worth time contributing since the odds are high that contributions are likely to be ignored. this results in the org being like "well no external people are contributing, so we may as well keep the same internal focused process"

**Comment by H-Tevin** [comment]
That's fair.
I'm sure both do a lot of work that goes unnoticed. CSH of course can do whatever they want, they're working for fun and anything they provide to students is an awesome bonus.
SG on the other hand, is tasked with supporting students and should be updating or at the very least maintaining their projects. Thought I do understand that nobody wants to maintain and fix bugs on some one else's project. They want to do new things, but they need to do a better job of explaining just what those other things are. Because right now, it seems to be very little.

Hopefully they start to allow outside contributions, because there are a lot of passionate students who want to help make RIT a better place to be.

**Comment by MoralCode** [comment]
> Hopefully they start to allow outside contributions, because there are a lot of passionate students who want to help make RIT a better place to be.

I know, right! I just submitted a paper to RIT's Strategic Planning Steering Committee to help them see the benefits of encouraging open source at a university level. Would love to connect with you and work with you to help get some cool stuff done on campus (i always have more projects than I do time)
